sys/fido2/doc.txt

```diff
@@ -102,6 +106,8 @@
  *
  * Abstraction for flash operations. Uses the RIOT [Flashpage MTD driver](http://api.riot-os.org/group__drivers__mtd__flashpage.html).
  *
+ * Flash memory is reserved at build time using the `FLASH_WRITABLE_INIT` macro. The amount of flashpages reserved can be configured as `FIDO2_CTAP_NUM_FLASHPAGES` in KConfig. The implementation needs atleast 1 flashpage to store state information and 1 flashpage to store credentials (rks). Therefore, the minimum amount of flashpages needed is 2. State information is stored on the first flashpage, credentials (rks) on the following flashpages.
```

Comment:
1. This line is too long
2. Can you introduce the abbreviation 'rk' here since it is the first occurance in the documentation?

Reply:
Is there a line length limit for doc.txt files ? Running `make doc` does not show any errors and the produced html also looks fine